brave/brave-core
# [Android] Add option to enable/disable download notification bubble

Resolves https://github.com/brave/brave-browser/issues/25611

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Update the "ask where to save files" copy to: "Ask where to save files (if SD card is detected)"
- Show download progress notifications with On/Off switch in download settings
- Enable the switch, it should show notification bubble when downloading file
- Disable the switch, it shouldn't show notification bubble when downloading file
